[2021-12-15 09:40:34,789] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_kerry.scrap_data scheduled__2021-12-14T09:40:00+00:00 [queued]>
[2021-12-15 09:40:34,994] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_kerry.scrap_data scheduled__2021-12-14T09:40:00+00:00 [queued]>
[2021-12-15 09:40:35,012] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-15 09:40:35,019] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-15 09:40:35,025] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-15 09:40:35,225] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-14 09:40:00+00:00
[2021-12-15 09:40:35,276] {standard_task_runner.py:52} INFO - Started process 14554 to run task
[2021-12-15 09:40:35,365] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_kerry', 'scrap_data', 'scheduled__2021-12-14T09:40:00+00:00', '--job-id', '2449', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_kerry.py', '--cfg-path', '/tmp/tmpdm5_7umv', '--error-file', '/tmp/tmpvhkuvuji']
[2021-12-15 09:40:35,376] {standard_task_runner.py:77} INFO - Job 2449: Subtask scrap_data
[2021-12-15 09:40:36,508] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_kerry.scrap_data scheduled__2021-12-14T09:40:00+00:00 [running]> on host e2904d252e94
[2021-12-15 09:40:38,142] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_kerry
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-14T09:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-14T09:40:00+00:00
[2021-12-15 09:40:38,393] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-15 09:40:38,454] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kerry
[2021-12-15 09:41:02,481] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-15 09:41:03,007] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-15 09:41:03,009] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-15 09:41:03,017] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-15 09:41:03,019] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-15 09:41:03,020] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-15 09:41:03,320] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-15 09:41:03,324] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-15 09:41:03,328] {spark_submit.py:514} INFO - deployMode              null
[2021-12-15 09:41:03,332] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-15 09:41:03,337] {spark_submit.py:514} INFO - executorCores           null
[2021-12-15 09:41:03,341] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-15 09:41:03,344] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-15 09:41:03,348] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-15 09:41:03,350] {spark_submit.py:514} INFO - driverCores             null
[2021-12-15 09:41:03,354] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-15 09:41:03,358] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-15 09:41:03,365] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-15 09:41:03,366] {spark_submit.py:514} INFO - supervise               false
[2021-12-15 09:41:03,367] {spark_submit.py:514} INFO - queue                   null
[2021-12-15 09:41:03,368] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-15 09:41:03,370] {spark_submit.py:514} INFO - files                   null
[2021-12-15 09:41:03,372] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-15 09:41:03,374] {spark_submit.py:514} INFO - archives                null
[2021-12-15 09:41:03,375] {spark_submit.py:514} INFO - mainClass               null
[2021-12-15 09:41:03,376] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-15 09:41:03,378] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-15 09:41:03,380] {spark_submit.py:514} INFO - childArgs               [kerry]
[2021-12-15 09:41:03,381] {spark_submit.py:514} INFO - jars                    null
[2021-12-15 09:41:03,383] {spark_submit.py:514} INFO - packages                null
[2021-12-15 09:41:03,384] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-15 09:41:03,386] {spark_submit.py:514} INFO - repositories            null
[2021-12-15 09:41:03,387] {spark_submit.py:514} INFO - verbose                 true
[2021-12-15 09:41:03,389] {spark_submit.py:514} INFO - 
[2021-12-15 09:41:03,390] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-15 09:41:03,392] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-15 09:41:03,393] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-15 09:41:03,394] {spark_submit.py:514} INFO - 
[2021-12-15 09:41:03,396] {spark_submit.py:514} INFO - 
[2021-12-15 09:41:04,966] {spark_submit.py:514} INFO - Main class:
[2021-12-15 09:41:04,968] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-15 09:41:04,970] {spark_submit.py:514} INFO - Arguments:
[2021-12-15 09:41:04,972] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-15 09:41:04,974] {spark_submit.py:514} INFO - null
[2021-12-15 09:41:04,976] {spark_submit.py:514} INFO - kerry
[2021-12-15 09:41:04,977] {spark_submit.py:514} INFO - --verbose
[2021-12-15 09:41:04,979] {spark_submit.py:514} INFO - Spark config:
[2021-12-15 09:41:04,981] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-15 09:41:04,984] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-15 09:41:04,985] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-15 09:41:04,987] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-15 09:41:04,988] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-15 09:41:04,989] {spark_submit.py:514} INFO - 
[2021-12-15 09:41:04,990] {spark_submit.py:514} INFO - 
[2021-12-15 09:41:04,991] {spark_submit.py:514} INFO - 
[2021-12-15 09:41:06,590] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-15 09:41:06,608] {spark_submit.py:514} INFO - 21/12/15 09:41:06 INFO SparkContext: Running Spark version 3.2.0
[2021-12-15 09:41:06,753] {spark_submit.py:514} INFO - 21/12/15 09:41:06 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-15 09:41:06,933] {spark_submit.py:514} INFO - 21/12/15 09:41:06 INFO ResourceUtils: ==============================================================
[2021-12-15 09:41:06,935] {spark_submit.py:514} INFO - 21/12/15 09:41:06 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-15 09:41:06,936] {spark_submit.py:514} INFO - 21/12/15 09:41:06 INFO ResourceUtils: ==============================================================
[2021-12-15 09:41:06,937] {spark_submit.py:514} INFO - 21/12/15 09:41:06 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-15 09:41:06,986] {spark_submit.py:514} INFO - 21/12/15 09:41:06 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-15 09:41:07,007] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO ResourceProfile: Limiting resource is cpu
[2021-12-15 09:41:07,008] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-15 09:41:07,138] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SecurityManager: Changing view acls to: ***
[2021-12-15 09:41:07,140] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SecurityManager: Changing modify acls to: ***
[2021-12-15 09:41:07,141] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SecurityManager: Changing view acls groups to:
[2021-12-15 09:41:07,143] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SecurityManager: Changing modify acls groups to:
[2021-12-15 09:41:07,144] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-15 09:41:07,659] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO Utils: Successfully started service 'sparkDriver' on port 46565.
[2021-12-15 09:41:07,701] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SparkEnv: Registering MapOutputTracker
[2021-12-15 09:41:07,752] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-15 09:41:07,784] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-15 09:41:07,785] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-15 09:41:07,793] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-15 09:41:07,829] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-0d00974b-834e-4ea8-9c4c-e1e4aedc7561
[2021-12-15 09:41:07,861] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-15 09:41:07,885] {spark_submit.py:514} INFO - 21/12/15 09:41:07 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-15 09:41:08,220] {spark_submit.py:514} INFO - 21/12/15 09:41:08 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-15 09:41:08,236] {spark_submit.py:514} INFO - 21/12/15 09:41:08 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2021-12-15 09:41:08,345] {spark_submit.py:514} INFO - 21/12/15 09:41:08 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://e2904d252e94:4041
[2021-12-15 09:41:08,872] {spark_submit.py:514} INFO - 21/12/15 09:41:08 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-15 09:41:09,156] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO TransportClientFactory: Successfully created connection to spark/172.19.0.4:7077 after 110 ms (0 ms spent in bootstraps)
[2021-12-15 09:41:09,378] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20211215094109-0076
[2021-12-15 09:41:09,396] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 43623.
[2021-12-15 09:41:09,398] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO NettyBlockTransferService: Server created on e2904d252e94:43623
[2021-12-15 09:41:09,402] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-15 09:41:09,418] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, e2904d252e94, 43623, None)
[2021-12-15 09:41:09,425] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO BlockManagerMasterEndpoint: Registering block manager e2904d252e94:43623 with 434.4 MiB RAM, BlockManagerId(driver, e2904d252e94, 43623, None)
[2021-12-15 09:41:09,430] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, e2904d252e94, 43623, None)
[2021-12-15 09:41:09,433] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, e2904d252e94, 43623, None)
[2021-12-15 09:41:09,967] {spark_submit.py:514} INFO - 21/12/15 09:41:09 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2021-12-15 09:41:10,066] {spark_submit.py:514} INFO - /home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py:238: FutureWarning: Python 3.6 support is deprecated in Spark 3.2.
[2021-12-15 09:41:10,068] {spark_submit.py:514} INFO - FutureWarning
[2021-12-15 09:41:10,578] {spark_submit.py:514} INFO - 21/12/15 09:41:10 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2021-12-15 09:41:10,621] {spark_submit.py:514} INFO - 21/12/15 09:41:10 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2021-12-15 09:41:13,961] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:14,670] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:15,383] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:15,808] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:16,537] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:17,179] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:18,622] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:19,649] {spark_submit.py:514} INFO - running...
[2021-12-15 09:41:20,794] {spark_submit.py:514} INFO - running...
[2021-12-15 09:46:57,110] {spark_submit.py:514} INFO - running...
[2021-12-15 09:46:59,456] {spark_submit.py:514} INFO - running...
[2021-12-15 09:47:00,829] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20211215094109-0076/0 on worker-20211215085029-172.19.0.5-33915 (172.19.0.5:33915) with 6 core(s)
[2021-12-15 09:47:00,832] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneSchedulerBackend: Granted executor ID app-20211215094109-0076/0 on hostPort 172.19.0.5:33915 with 6 core(s), 1024.0 MiB RAM
[2021-12-15 09:47:00,846] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20211215094109-0076/1 on worker-20211215085029-172.19.0.6-38383 (172.19.0.6:38383) with 6 core(s)
[2021-12-15 09:47:00,854] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneSchedulerBackend: Granted executor ID app-20211215094109-0076/1 on hostPort 172.19.0.6:38383 with 6 core(s), 1024.0 MiB RAM
[2021-12-15 09:47:00,980] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20211215094109-0076/1 is now LOST (worker lost)
[2021-12-15 09:47:00,985] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneSchedulerBackend: Executor app-20211215094109-0076/1 removed: worker lost
[2021-12-15 09:47:00,990] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneAppClient$ClientEndpoint: Master removed worker worker-20211215085029-172.19.0.6-38383: Not receiving heartbeat for 60 seconds
[2021-12-15 09:47:00,997] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneSchedulerBackend: Worker worker-20211215085029-172.19.0.6-38383 removed: Not receiving heartbeat for 60 seconds
[2021-12-15 09:47:00,998] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20211215094109-0076/0 is now LOST (worker lost)
[2021-12-15 09:47:01,000] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneSchedulerBackend: Executor app-20211215094109-0076/0 removed: worker lost
[2021-12-15 09:47:01,001] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneAppClient$ClientEndpoint: Master removed worker worker-20211215085029-172.19.0.5-33915: Not receiving heartbeat for 60 seconds
[2021-12-15 09:47:01,003] {spark_submit.py:514} INFO - 21/12/15 09:47:00 INFO StandaloneSchedulerBackend: Worker worker-20211215085029-172.19.0.5-33915 removed: Not receiving heartbeat for 60 seconds
[2021-12-15 09:47:01,025] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO BlockManagerMaster: Removal of executor 1 requested
[2021-12-15 09:47:01,027] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asked to remove non-existent executor 1
[2021-12-15 09:47:01,031] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO TaskSchedulerImpl: Handle removed worker worker-20211215085029-172.19.0.6-38383: Not receiving heartbeat for 60 seconds
[2021-12-15 09:47:01,036] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO BlockManagerMaster: Removal of executor 0 requested
[2021-12-15 09:47:01,045] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asked to remove non-existent executor 0
[2021-12-15 09:47:01,051] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO TaskSchedulerImpl: Handle removed worker worker-20211215085029-172.19.0.5-33915: Not receiving heartbeat for 60 seconds
[2021-12-15 09:47:01,103] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO BlockManagerMasterEndpoint: Trying to remove executor 1 from BlockManagerMaster.
[2021-12-15 09:47:01,108] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO BlockManagerMasterEndpoint: Trying to remove executor 0 from BlockManagerMaster.
[2021-12-15 09:47:01,122] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO DAGScheduler: Shuffle files lost for worker worker-20211215085029-172.19.0.6-38383 on host 172.19.0.6
[2021-12-15 09:47:01,130] {spark_submit.py:514} INFO - 21/12/15 09:47:01 INFO DAGScheduler: Shuffle files lost for worker worker-20211215085029-172.19.0.5-33915 on host 172.19.0.5
[2021-12-15 09:47:01,205] {local_task_job.py:212} WARNING - State of this instance has been externally set to up_for_retry. Terminating instance.
[2021-12-15 09:47:01,211] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 14554
[2021-12-15 09:47:01,214] {taskinstance.py:1411} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-12-15 09:47:01,217] {spark_submit.py:644} INFO - Sending kill signal to spark-submit
[2021-12-15 09:47:01,363] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 433, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 482, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1413, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2021-12-15 09:47:01,439] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_kerry, task_id=scrap_data, execution_date=20211214T094000, start_date=20211215T094034, end_date=20211215T094701
[2021-12-15 09:47:01,498] {standard_task_runner.py:91} ERROR - Failed to execute job 2449 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 433, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 482, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1413, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2021-12-15 09:47:01,559] {process_utils.py:66} INFO - Process psutil.Process(pid=14554, status='terminated', exitcode=1, started='09:40:34') (14554) terminated with exit code 1
[2021-12-15 09:47:01,564] {process_utils.py:66} INFO - Process psutil.Process(pid=14636, status='terminated', started='09:41:04') (14636) terminated with exit code None
[2021-12-15 09:47:01,571] {process_utils.py:66} INFO - Process psutil.Process(pid=14555, status='terminated', started='09:40:37') (14555) terminated with exit code None
